athenaCL/libATH/command.py

```diff
@@ -1517,7 +1515,7 @@ def gather(self):
             self.confirm = 'confirm'
 
             # only do this if user has interactively confirmed quit
-            if self.ao.external.logCheck() == 1: # log exists, submit
+            if self.ao.external.logCheck() == 0: # log exists, submit
```

Comment:
Could you please explain this change?

Reply:
Hi Ales,
i made the change: 
if self.ao.external.logCheck() == 0: # log exists, submit
because every time i quit AthenaCL, it asks to submit anonymously a bug report to Christopher Ariza.
He is not actively maintaining it anymore,  i found it to be annoying and redundant. 
I found this switch so i turned it off.

Reply:
> because every time i quit AthenaCL, it asks

Ah, ok. So when there were some errors, while you were using the app, athenaCL asks you to send the log when you quit. Then we should update the bug report method instead. For example, we can suggest opening an issue in this repo and provide the path to the log, or print it to stdout, for example. Otherwise, if it asks you to send the report even if there were no errors, it's a bug. Could you, please, open an issue for this?

Meanwhile, please return the previous value back.